src/DonationForms/resources/registrars/templates/fields/Gateways.tsx

```diff
@@ -2,6 +2,23 @@ import {ErrorMessage} from '@hookform/error-message';
 import type {GatewayFieldProps, GatewayOptionProps} from '@givewp/forms/propTypes';
 import {ErrorBoundary} from 'react-error-boundary';
 import {__} from '@wordpress/i18n';
+import {useEffect, useMemo} from 'react';
+
+function GatewayMissingMessage({currencyNotSupported}: {currencyNotSupported?: boolean}) {
```

Comment:
This file doesn't have any `unreleased` tag

Reply:
Resolved 9d89af22